etn-ccis/blui-doc-it
# Update dependencies

## Changes proposed in this Pull Request:

- Update dependencies to the latest version to get the latest icons

#### To Test:

- Verify that icons we added [in this PR](https://github.com/etn-ccis/blui-icons/pull/269) is available in the icons page
